api/bazel/api_build_system.bzl

```diff
@@ -98,6 +111,7 @@ def api_proto_library(
         has_services = 0,
         linkstatic = None,
         require_py = 1):
+    this = ":" + name
```

Comment:
s/this/relative_name/

Reply:
Done in 2740d3eb6924e5aa3105b456f9438441fe3be6d4